test/wg-easy/upgrade_strategy

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+import json
+import sys
+import re
+
+from catalog_update.upgrade_strategy import datetime_versioning
+
+
+# Upstream uses only a single digit for tags
+VERSION_REG = r'[0-9]*'
```

Comment:
I assume this has been copied over ;) The upgrade strategy will fail to get latest tags - i think we can remove this for now and add it later separately in a subsequent PR.

Reply:
I've copied an update_strategy script and modified the regex to match single digit for tags (thats what wg-easy uses)
If you think it won't work, yea I can remove it for now!

Reply:
Yup it wont as the newer mapping function below was for a different application. We can go through this later in a subsequent PR